codegen/method.go

```diff
@@ -1000,6 +1003,55 @@ func (ms *MethodSpec) setTypeConverters(
 	return nil
 }
 
+func (ms *MethodSpec) setDummyTypeConverters(
+	funcSpec *compile.FunctionSpec,
+	reqTransforms map[string]FieldMapperEntry,
+	headersPropagate map[string]FieldMapperEntry,
+	respTransforms map[string]FieldMapperEntry,
+	reqrespTransforms map[string]FieldMapperEntry,
+	h *PackageHelper,
+) error {
+
+	dummyConverter := NewTypeConverter(h, nil)
+
+	respType := funcSpec.ResultSpec.ReturnType
+
+	dummyConverter.append(
+		"func convert",
+		PascalCase(ms.Name),
+		"DummyResponse(in ", ms.RequestType, ") ", ms.ResponseType, "{")
+
+	structType := compile.FieldGroup(funcSpec.ArgsSpec)
+
+	switch respType.(type) {
+	case
+		*compile.BoolSpec,
+		*compile.I8Spec,
+		*compile.I16Spec,
+		*compile.I32Spec,
+		*compile.EnumSpec,
+		*compile.I64Spec,
+		*compile.DoubleSpec,
+		*compile.StringSpec:
+
+		dummyConverter.append("out", " := in\t\n")
+	default:
+		// default as struct
```

Comment:
can it not be list/set/map?
Do you want a set of cases to either eliminate or covert those?

Reply:
List,sets and maps would default to structs and get converted using the type converter, i.e. it will be a deep copy. I will try and add endpoint test cases for these.